indico/modules/events/editing/client/js/components/ReviewConditionsManager.jsx

```diff
@@ -0,0 +1,104 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2020 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import reviewConditionsURL from 'indico-url:event_editing.api_review_conditions';
+
+import React, {useState} from 'react';
+import PropTypes from 'prop-types';
+import {Button, Divider, Loader, Message} from 'semantic-ui-react';
+
+import {handleSubmitError} from 'indico/react/forms';
+import {useIndicoAxios} from 'indico/react/hooks';
+import {Translate} from 'indico/react/i18n';
+import {indicoAxios} from 'indico/utils/axios';
+
+import ConditionInfo from './ConditionInfo';
+import ReviewConditionForm from './ReviewConditionForm';
+
+import './ReviewConditionsManager.module.scss';
+
+export default function ReviewConditionsManager({eventId, fileTypes}) {
+  const [isAdding, setIsAdding] = useState(false);
+  const {loading, reFetch, data: eventConditionsSetting, lastData} = useIndicoAxios({
+    url: reviewConditionsURL({confId: eventId}),
+    trigger: eventId,
+  });
+
+  const eventConditions = eventConditionsSetting || lastData;
+  if (loading && !lastData) {
+    return <Loader inline="centered" active />;
+  } else if (!eventConditions) {
+    return null;
+  }
+
+  const reviewConditions = eventConditions.map(condition =>
```

Comment:
I think this function could use a comment... it's not very obvious what it's doing (replacing the ID with the actual file type?)